### PR TITLE
Fix scrolling for new design

### PR DESCRIPTION
**Pull request by artonge**
Signed-off-by: Louis Chemineau <louis@chmn.me>

### CODE REVIEW

**Comment by marcelklehr** [comment]
TagContent and floders views are also broken. Not sure about On this day, favorites, people and shared with you

File: src/views/AlbumContent.vue [hunk 1]
```diff
@@ -530,14 +531,13 @@ export default {
 	}
 
 	&__photos {
-		height: 100%;
+		height: calc(100% - 60px);
```
Comment (marcelklehr) [inline src/views/AlbumContent.vue:534]: This makes FileListViewer the scrollable element, but shouldn't the app content be the scrollable element?

Comment (marcelklehr) [reply src/views/AlbumContent.vue:534]: Ah, we probably need vue v7 for that, right?

Comment (artonge) [reply src/views/AlbumContent.vue:534]: At this point, we don't really care which element is scrollable, no ?

**Comment by artonge** [comment]
Superseded by https://github.com/nextcloud/photos/pull/1206

**Pull request closed without merging**